src/Grpc/test/InteropTests/Helpers/ClientProcess.cs

```diff
@@ -52,6 +62,23 @@ private void Process_OutputDataReceived(object sender, DataReceivedEventArgs e)
                 {
                     _startTcs.TrySetResult(null);
                 }
+
+                lock (_outputLock)
+                {
+                    _output.AppendLine(data);
+                }
+            }
+        }
+
+        private void Process_ErrorDataReceived(object sender, DataReceivedEventArgs e)
+        {
+            var data = e.Data;
+            if (data != null)
+            {
+                lock (_outputLock)
+                {
+                    _output.AppendLine(data);
```

Comment:
Should error and regular output be distinguished in the buffer e.g. with an `ERROR: ` prefix?

Reply:
The only error output would be if an exception is thrown, and it should be obvious. Will change in the future if it is confusing.